lib/common/utils.js

```diff
@@ -299,6 +300,7 @@ exports.lib                 = {};
 exports.lib.mdurl           = require('mdurl');
 exports.lib.ucmicro         = require('uc.micro');
 
+exports.isNil               = isNil;
```

Comment:
It's not good idea to spread micro helpers as lib feature, when used in only one place. Such things clutter api significantly in long term.

Reply:
Was trying to follow the existing organization.

Where do you want it?

Reply:
Until "helper" is not used wide, it should be in the same file where it's needed. Preferably - inlined, without separate fn at all.